data/hai/hai missions.txt

```diff
@@ -648,7 +648,7 @@ mission "Unwanted Cargo"
 	landing
```

Comment:
```suggestion
	minor
	landing
```
While we're here, thoughts on adding `minor` to the mission? This coming up in the middle of a multi-mission story like a campaign feels like it could lead to some pacing interruption to me.

Reply:
iirc minor don't work with landing.